### ISSUE TITLE
Documentation: IE 11 Polyfill Required for Uint8Array

### ISSUE THREAD

**Issue by rcollette** [okta/okta-auth-js#359]
I'm using the Okta sign in widget with PKCE

I was getting an error:
```
TypeError: Object doesn't support property or method 'slice'
```

I found that I had to add a polyfill for Uint8Array to make it work.

```
// Text encoding required for Edge and IE 11 to support Okta sign in widget.
import 'fast-text-encoding';
// Webcrypto-shim required for IE11 to support Okta sign in widget.
import 'webcrypto-shim';
// Array slice required for IE 11 to support Okta sign in widget.
import 'core-js/modules/es6.array.from';
// Typed array support required for IE 11 to support Okta sign in widget.
import 'core-js/modules/es6.typed.uint8-array';
```

Unless I'm missing something, there seems to be a gap in the documentation.

**Comment by swiftone** [comment]
Internal note: revisit after OKTA-290745

**Comment by swiftone** [comment]
Internal ref: OKTA-291509

**Comment by SharonQSY** [comment]
Thanks @rcollette. It saved my day.

I used okta-angular library for the application. As the application contains `imports: [RouterModule.forRoot(routes, { useHash: true })]`, I made the changed based on the comments in https://github.com/okta/okta-oidc-js/issues/672.

Chrome, Firefox, and edge are working fine. However, IE has looping issue. 
Thanks for your comments, I added `import 'core-js/es/typed-array';` fixed the issue.
